src/validation/rules/NoFragmentCycles.js

```diff
@@ -34,9 +34,7 @@ export function NoFragmentCycles(context: ValidationContext): ASTVisitor {
   return {
     OperationDefinition: () => false,
     FragmentDefinition(node) {
-      if (!visitedFrags[node.name.value]) {
-        detectCycleRecursive(node);
-      }
+      detectCycleRecursive(node);
```

Comment:
Do you mind factoring the changes to this file to a separate PR?

Reply:
@leebyron Done as #1381